pkg/genericapiserver/authorizer/authz.go

```diff
@@ -72,6 +72,29 @@ func NewAlwaysFailAuthorizer() authorizer.Authorizer {
 	return new(alwaysFailAuthorizer)
 }
 
+type privilegedGroupAuthorizer struct {
+	user string
+	uid  string
+}
+
+func (r *privilegedGroupAuthorizer) Authorize(attr authorizer.Attributes) (bool, string, error) {
+	if r.user != "" && attr.GetUser() != nil &&
+		attr.GetUser().GetName() == r.user &&
+		attr.GetUser().GetUID() == r.uid {
+		return true, "", nil
+	}
+	return false, "", errors.New("Not in privileged list.")
```

Comment:
Return `"Not in privileged list"` as the `string` and `nil` as the error.  You didn't fail during evaluation, you definitively denied them access for a particular reason.